CHANGELOG.md

```diff
@@ -22,10 +22,13 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
   (defaults to `true`)
   - `DOCKER_VERNEMQ_ASTARTE_VMQ_PLUGIN__CASSANDRA__SSL_CUSTOM_SNI`
   - `DOCKER_VERNEMQ_ASTARTE_VMQ_PLUGIN__CASSANDRA__SSL_CA_FILE`
+  - `DOCKER_VERNEMQ_ASTARTE_VMQ_PLUGIN__ASTARTE_INSTANCE_ID`
+  (defaults to `default`)
 - Added support for device deletion. During deletion, a device is
   disconnected and not allowed to reconnect until deletion ends.
   Inflight messages are discarded. After deletion, a device must be
   registered again in order to connect to Astarte.
+- Edited queries to work with keypaces separation
```

Comment:
This line could be a bit obscure if someone is not into the Astarte process. Something like "Added support for multiple Astarte instances sharing the same database" is more user-friendly.